tests/monotouch-test/Network/NWPathMonitorTest.cs

```diff
@@ -0,0 +1,117 @@
+﻿#if !__WATCHOS__
+using System;
+using System.Threading;
+using Network;
+using CoreFoundation;
+using Foundation;
+
+using NUnit.Framework;
+
+
+namespace monotouchtest.Network
+{
+	[TestFixture]
+	[Preserve(AllMembers = true)]
```

Comment:
Looks like you're spaces->tabs conversion broke the coding style, e.g. space before `(`, across all the file.
Double check your VSfM settings - that _should_ not happen